<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>System_Setting_Form</class>
 <widget class="QWidget" name="System_Setting_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>642</width>
        <height>428</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Channel">
          <attribute name="title">
           <string>Channel</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>The Channel Number</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QSpinBox" name="ChannelNumber">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="frame">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="prefix">
                 <string/>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>199</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="SaveImage_1">
             <property name="title">
              <string>Save Image One</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="1">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Path Format</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="CheckPathPushButton">
                <property name="text">
                 <string>Check Path</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="ImageFormat">
                <property name="editable">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>CHANNEL/YYYY/MM/DD/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CHANNEL/YYYY/MM/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CHANNEL/YYYY/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CHANNEL/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YYYY/MM/DD/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YYYY/MM/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>YYYY/</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>/</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLineEdit" name="ImgPathlineEdit">
                <property name="placeholderText">
                 <string>/home/cc/</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="ImageNaming">
             <property name="title">
              <string>Image naming rules</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QComboBox" name="ImageNamingRules">
                <property name="editable">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>DateTime-Channel-Number</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DateTime-Number-Channel</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DateTime-Result</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>50</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Recognizer">
          <attribute name="title">
           <string>Recognizer</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Check pattern</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <property name="leftMargin">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QRadioButton" name="ColorDisplay">
                <property name="text">
                 <string>Color display</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="AutomaticCorrection">
                <property name="text">
                 <string>Automatic correction</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Identify screening</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLineEdit" name="Contains_lineEdit">
                  <property name="placeholderText">
                   <string>WDFU</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Contains_pushButton">
                  <property name="text">
                   <string>Contains</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLineEdit" name="Eliminate_lineEdit">
                  <property name="placeholderText">
                   <string>CCAU</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="Eliminate_pushButton">
                  <property name="text">
                   <string>Eliminate</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>178</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Upload">
          <attribute name="title">
           <string>Upload</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="1" column="0">
            <widget class="QCheckBox" name="ReduceImage">
             <property name="text">
              <string>Reduce image</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>15</width>
               <height>53</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="Ftp">
             <property name="title">
              <string>Ftp</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="1">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLineEdit" name="FtpUser"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="FtpPassword"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="FtpAddress"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="FtpPort"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="FtpRemote"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="FtpLocal"/>
                </item>
               </layout>
              </item>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>                            User:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>                    Password:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>                       Address:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>                              Port:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>The remote directory:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>            Local directory:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Service">
          <attribute name="title">
           <string>Service</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_13">
             <property name="title">
              <string>Socket Type</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_25">
              <item row="1" column="0">
               <widget class="QStackedWidget" name="Service_Type_stackedWidget">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_5">
                 <layout class="QGridLayout" name="gridLayout_11">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QLineEdit" name="Address_Singleton_lineEdit">
                    <property name="placeholderText">
                     <string>127.0.0.1:5000</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <spacer name="verticalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="page_6">
                 <layout class="QGridLayout" name="gridLayout_28">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item row="0" column="0">
                   <widget class="QTextEdit" name="Address_Many_textEdit">
                    <property name="placeholderText">
                     <string>if(server)192.168.1.100:5000,192.168.1.100:5001,192.168.1.100:5002 if(client)192.168.1.100:5000,192.168.1.101:5000,192.168.1.102:5000</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QComboBox" name="Service_Type_comboBox">
                <item>
                 <property name="text">
                  <string>One service corresponds to all channels (Need to respond to channel instruction [L|Number])</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>One service corresponds to one channel</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="Resulting_checkBox">
             <property name="text">
              <string>Only the resulting data is sent</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="Hearbeat_checkBox">
             <property name="text">
              <string>The heartbeat packets</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_10">
             <property name="title">
              <string>Socket Model</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_26">
              <item row="0" column="1">
               <widget class="QRadioButton" name="ClientModel">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Tcp Client</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="ServerModel">
                <property name="text">
                 <string>Tcp Server</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="Disconnected_checkBox">
             <property name="text">
              <string>Send completion is actively disconnected</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Other</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_11">
             <property name="title">
              <string>Log</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_20">
              <item row="0" column="0">
               <widget class="QCheckBox" name="InfoLog">
                <property name="text">
                 <string>save info log</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="DebugLog">
                <property name="text">
                 <string>save debug log</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Select the language</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="0" column="0">
               <widget class="QComboBox" name="Language">
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>English</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Chinese</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>196</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Start</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_16">
              <item row="0" column="0">
               <widget class="QCheckBox" name="StartupMinimization">
                <property name="text">
                 <string>Startup minimization</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="AutomaticStart">
                <property name="text">
                 <string>Automatic start</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Service_Type_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Service_Type_stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
